graphql_api/dataloader/bundle_analysis.py

```diff
@@ -33,3 +33,17 @@ def load_bundle_analysis_comparison(
         base_report_key=base_report.external_id,
         head_report_key=head_report.external_id,
     )
+
+
+def load_bundle_analysis_report(commit: Commit) -> BundleAnalysisReport:
+    report = CommitReport.objects.filter(
+        report_type=CommitReport.ReportType.BUNDLE_ANALYSIS, commit=commit
+    ).first()
+    if report is None:
+        return MissingHeadReport()
```

Comment:
Could this fn ever load a non-head report? Trying to see if it's better to say MissingReport as opposed of the head

Reply:
Doesn't appear like we have anything more suitable. FWIW we do use `MissingHeadReport` on non comparison like Commits.pathContents